scripts/run_k4_overnight_sweeps.py

```
@@ -0,0 +1,34 @@
+
+from kryptos.k4.overnight_runner import run_all_pending_sweeps
+
+
+def _progress(name: str, status: str) -> None:
+    print(f"[{name}] {status}", flush=True)
+
+
+def main() -> int:
+    result = run_all_pending_sweeps(progress_cb=_progress)
+    if result["status"] == "breakthrough":
+        print(f"\n*** BREAKTHROUGH in sweep '{result['breakthrough_sweep']}' -- see snapshot path above ***")
```

Pylint (4.0.7)

[convention] Line too long (109/88)

(C0301)